core/event/mutable-event.js

```diff
@@ -236,6 +236,13 @@ if (typeof window !== "undefined") {
         touches: {
             get: function () {
                 return this._event.touches;
+            },
+            set: function (value) {
+                if (Array.isArray(value) && this._event.touches) {
+                    this._event.touches.splice.apply(this._event.touches, [0, Infinity].concat(value));
+                } else if (Array.isArray(value)) {
+                    this._event.touches = value;
+                }
             }
```

Comment:
if (Array.isArray(value) could be done twice, maybe worth refactoring to avoid, same in changedTouches

Reply:
@marchant resolved in both setters